### ISSUE TITLE
stick (mouse) drift in 3.3.4 only

### ISSUE THREAD

**Issue by bluenote73** [AntiMicroX/antimicrox#808]
### Is there an existing issue for this?

- [X] I searched the existing issues and did not find anything similar.

### Current Behavior

Using the same profile, mouse movement "sticks" in a particular direction, and gets more and more severe. But the cause seems random. When it occurs to me it is exacerbated by activating right trigger (which is also a set change, and back on release). The "stuckness" can be reset by moving the stick. Which then no longer acts stuck, unless I use the trigger/set change again, then goes through the same progression of getting worse and worse until reset. 

Reinstalling 3.3.3 restores proper functionality. 

I have reinstalled 3.3.4 and 3.3.3 twice to confirm.

### Expected Behavior

Stick movement shouldn't continue when it isn't being activated. Nor should the range of extension increase.

### Steps To Reproduce

It triggers randomly but quickly. 3.3.4 is unusable for me.

### Environment

```markdown
Program Version 3.3.4
Compiled from packaging: GitHub Windows Release
Built Against SDL 2.26.5
Running With SDL 2.26.5
Using Qt 5.15.2
Using Event Handler: SendInput
Host OS: windows Version: 10 Architecture: x86_64
```

### Anything else?

I can upload profile file if need be

**Comment by pktiuk** [comment]
As in many cases,  I think it may be caused by the regression in the SDL between 3.3.3 and 3.3.4 there was an update of SDL version. https://github.com/AntiMicroX/antimicrox/pull/771

I think you could easily verify this by following these steps (using portable app versions would be easier than reinstalling):

1. Download 3.3.4 and 3.3.3 portable releases of antimicrox
2. Copy SDL2.dll file from 3.3.4 directory to 3.3.3 directory (replace existing one).
3. Run AnntimicroX in 3.3.3 version

Now in the info-> about window you should see something like:

```
Program Version 3.3.3
Compiled from packaging: GitHub Windows Release
Built Against SDL 2.0.20
Running With SDL 2.26.5
```
This will mean, that 3.3.3 runs with updated SDL.

4. Test it. If 3.3.3 started having the same issues as 3.3.4 then it is a bug in the SDL, and it should be reported its developers.

Could you do it?

**Comment by bluenote73** [comment]
Yes, I was able to confirm that the problem follows the newer SDL file. I confirmed it in both directions. Old SDL with 333 and 334 works fine. New SDL has the problem on both.

**Comment by pktiuk** [comment]
@bluenote73    
In this case you have to report it in SDL repository https://github.com/libsdl-org/SDL/issues (and mention this issue in the description).

**Comment by bluenote73** [comment]
FYI they closed my issue as duplicate, then never did anything with the other reports, asking for them to be updated before they going to not delete them. they seem to be a very bad project, forcing people to keep submitting is silly, esp when its end users who have to wait for a new release of whatever end program they use. 
I am quite motivated to migrate off your software because of SDL's attitudes.
